### ISSUE TITLE
pinnedRight adding an extra blank column

### ISSUE THREAD

**Issue by AnirudhaGohokar** [angular-ui/ui-grid#4949]
Hi,

Setting `pinnedRight` to true is adding an extra blank column to its left.
Here's a plunker for reference: http://plnkr.co/edit/h3TzNPIhFp6BsV4C3PHE?p=preview

Thanks,
Anirudha

**Comment by zakerby** [comment]
Hi,
I have the same issue, it can be quick fixed by modifying the margin of the concerned column but it's not quite elegant....

**Comment by mani301** [comment]
Hi,
I did solved this problem by making certain changes in the **updateColumnWidths** function of **GridRenderContainer**. All I have done is added a new variable to find out if there is atleast one pinnedRightColumn is available. If its there, I have just added the scrollbarWidth(which is the exact size of whitespace / extra blank column) to the available width before **processColumnUpwards** and **processColumnDownwards** so that white space will be filled up with the columns during auto width specification.

//This is inside iterating for each column
`if (!pinRightColumn && column.colDef.pinnedRight) {
	pinRightColumn = true;
}`
//Before 
`if(pinRightColumn) {
	availableWidth += self.grid.scrollbarWidth;
}`

I am not quite sure this is correct solution or not, but this does the trick for me and has overwritten that API into my project. 

Adding my comment to know whether this is proper fix or not. Can anyone validate this approach, so that I can make a pull request into master.
